schemas/stsci.edu/asdf/transform/remap_axes-1.3.0.yaml

```diff
@@ -0,0 +1,91 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/transform/remap_axes-1.3.0"
+tag: "tag:stsci.edu:asdf/transform/remap_axes-1.3.0"
+title: >
+  Reorder, add and drop axes.
+
+description: |
```

Comment:
This description needs to be updated, now that the option to use a constant has been removed.